_posts/en/newsletters/2023-12-20-newsletter.md

```diff
@@ -0,0 +1,986 @@
+[apr bitgo]: /en/bitgo-musig2/
+[apr blinding]: /en/newsletters/2023/04/05/#bolts-765
+[apr htlcendo]: /en/newsletters/2023/05/17/#testing-htlc-endorsement
+[apr ldk]: /en/newsletters/2023/04/26/#ldk-0-0-115
+[apr lnd]: /en/newsletters/2023/04/05/#lnd-v0-16-0-beta
+[apr loop]: /en/newsletters/2023/05/24/#lightning-loop-defaults-to-musig2
+[apr musig2]: /en/newsletters/2023/04/12/#bips-1372
+[apr musig2 psbt]: /en/newsletters/2023/10/18/#proposed-bip-for-musig2-fields-in-psbts
+[apr rgb]: /en/newsletters/2023/04/19/#rgb-update
+[apr secp]: /en/newsletters/2023/04/12/#libsecp256k1-0-3-1
+[apr signrpc]: /en/newsletters/2023/02/15/#lnd-7171
+[apr splicing]: /en/newsletters/2023/04/12/#splicing-specification-discussions
+[apr taproot channels]: /en/newsletters/2023/08/30/#lnd-7904
+[apr watchtower]: /en/newsletters/2023/04/05/#watchtower-accountability-proofs
+[aug antidos]: /en/newsletters/2023/08/09/#denial-of-service-dos-protection-design-philosophy
+[aug bdk]: /en/newsletters/2023/08/09/#bdk-0-28-1
+[aug btcpay]: /en/newsletters/2023/08/02/#btcpay-server-1-11-1
+[aug cln]: /en/newsletters/2023/08/30/#core-lightning-23-08
+[aug combo]: /en/newsletters/2023/08/30/#covenant-mashup-using-txhash-and-csfs
+[aug fraud]: /en/newsletters/2023/08/23/#fraud-proofs-for-outdated-backup-state
+[aug fundingdos]: /en/newsletters/2023/08/30/#disclosure-of-past-ln-vulnerability-related-to-fake-funding
+[aug htlcendo]: /en/newsletters/2023/08/09/#htlc-endorsement-testing-and-data-collection
+[aug lnd taproot]: /en/newsletters/2023/08/30/#lnd-7904
+[aug milksad]: /en/newsletters/2023/08/09/#libbitcoin-bitcoin-explorer-security-disclosure
+[aug onion]: /en/newsletters/2023/08/09/#bolts-759
+[aug opreturn]: /en/newsletters/2023/08/09/#proposed-changes-to-bitcoin-core-default-relay-policy
+[aug payjoin]: /en/newsletters/2023/08/16/#serverless-payjoin
+[chatbtc]: https://chat.bitcoinsearch.xyz/
+[dec bcc]: /en/newsletters/2023/12/06/#bitcoin-core-26-0
+[dec cluster]: /en/newsletters/2023/12/06/#cluster-mempool-discussion
+[dec liqad]: /en/newsletters/2023/12/13/#discussion-about-griefing-liquidity-ads
+[dec payjoin]: /en/newsletters/2023/12/13/#payjoin-client-for-bitcoin-core-released
+[dec warnet announce]: /en/newsletters/2023/12/13/#bitcoin-network-simulation-tool-warnet-announced
+[dec zipkin warnet]: /en/newsletters/2023/12/06/#testing-with-warnet
+[feb bitcoinsearch]: /en/newsletters/2023/02/15/#bitcoinsearch-xyz
+[feb cln offers]: /en/newsletters/2023/02/08/#core-lightning-5892
+[feb codex32]: /en/newsletters/2023/02/22/#proposed-bip-for-codex32-seed-encoding-scheme
+[feb eclair offers]: /en/newsletters/2023/02/22/#eclair-2479
+[feb htlcendo]: /en/newsletters/2023/02/22/#feedback-requested-on-ln-good-neighbor-scoring
+[feb lnflag]: /en/newsletters/2023/02/22/#ln-quality-of-service-flag
+[feb miniscript]: /en/newsletters/2023/02/22/#bitcoin-core-24149
+[feb op_vault2]: /en/newsletters/2023/03/08/#alternative-design-for-op-vault
```

Comment:
This topic was described in a March newsletter and the source email is from March 1st.

Reply:
Yeah, but I think we described it in the February section.  The month names in these are just to reduce the chance of a accidentally redefinition of a previously used link.